app/models/cfp/proposal.rb

```diff
@@ -9,7 +9,7 @@ class Proposal < ActiveRecord::Base
     belongs_to :user, class_name: "::User"
     has_many :comments
     has_many :ranks
-    scope :approved, where(:approved => true)
+    scope :approved, -> { where(:approved => true) }
```

Comment:
I think you can use the new  syntax  for the hashes:
scope :approved, -> { where( approved: true) }